### PR TITLE
posts: QtWebEngine on Raspberry Pi 3 blogpost

### PR DESCRIPTION
**Pull request by GordanM**

### CODE REVIEW

**Comment by GordanM** [comment]
@FlorentRevest I guess you can just pick this pull request and finish it up a bit :). That way we can close the user story. Does that sound fine with you?

**Comment by FlorentRevest** [comment]
@GordanM thanks for the template. Sure, will do!

File: _posts/2018-04-04-QtWebEngine-on-Rpi3.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title:  "QtWebEngine on Raspberry Pi 3 using a Wayland client"
+date:   2018-04-04 19:00:00 +0100
+categories: yocto qt
+author: Florent Revest
+---
+
+[Netpune UI's] (https://doc-snapshots.qt.io/neptuneui/index.html)
+Browser Application is created using the QtWebEngine, while one off
```
Comment (tobsan) [inline _posts/2018-04-04-QtWebEngine-on-Rpi3.md]: s/off/of

**Comment by rpannek** [comment]
Please use `WIP:` in the headline of the pull request if you don't want people to spend time reviewing it yet.

**Comment by GordanM** [comment]
Florent is working on it #70 .

**Pull request closed without merging**